### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by TurkrosoftLabs** [alichherawalla/offline-mobile-llm-manager#191]
Consider using native controls for UI elements. The sliders where the user can adjust various parameters under the Settings tab, cannot be selected/interacted with using VoiceOver, the screen reader for iOS and other Apple platforms. VoiceOver also speaks weird unicode symbols on certain items, so they should be labeled properly. One such example is "" (Back) to go back from a subcategory under the Settings tab.

**Comment by alichherawalla** [comment]
thats a good point, will add support
